### ISSUE TITLE
Think you have a perf/memory issue in your .NET OSS project or application?

### ISSUE THREAD

**Issue by maartenba** [maartenba/maartenba.github.io#3]
**TL;DR:** Think you have a perf/memory issue in your .NET OSS project or application? Comment below and I *may* help you out (if I can blog/talk about the experience if we find something)

## Think you have a perf/memory issue in your OSS project or application?

I am trying to gather material for a blog post (and probably a talk / screencast at JetBrains) around performance and/or memory issues in open source projects or applications. Ideally, I build this material by also helping out on that project and fixing the issues discovered, if possible.

Therefore I am reaching out to you, .NET OSS project maintainers! If you feel you have a performance or memory issue in your project or application and can describe where you suspect something to be wrong (so we can do a more targeted search), please let me know in the comments below.

### Information required

Information required in your comment:

* Project GitHub URL
* Description of the issue you see (or think you may have)

### Which projects will I help out? (disclaimer)

Depending on the number of responses, I can probably not help out everyone. Ideally, the OSS project is well-known in the community (hey, I want eyeballs on the writeup afterwards), but if the issue is intrigueing that just may not matter. The main driver will be being able to blog/talk about the experience if we find something.

The selection process will not be transparent, and subject to my own interests and estimated effort vs. time possibly spent on the couch/with family/...
